### DISCUSSION TITLE
Sediakan fitur untuk  pendataan dan penyaluran santunan

### DISCUSSION THREAD

**Discussion by utay-cugung** [buku-masjid/buku-masjid#74] [Ideas]
1. Untuk memudahkan penyaluran santunan kepada anak yatim piatu terutama pada agenda 10 muharam atau hari lebarannya anak yatim piatu.

Penginputan bisa di buat sederhana saja (opsional) dengan menu penulisan termasuk judul:
- nama
- bin/binti
- tanggal lahir > untuk hitung umur otomatis
- jenis kelamin
- alamat rt/rw/dusun

2. Atau bisa juga di buat fleksibel agar bisa dibuat untuk pendataan fakir miskin atau bisa juga penerima zakat.

**Comment by nafiesl** [comment]
mas @utay-cugung untuk usulan ini, kita belum ada rencana ke arah sini dalam waktu dekat. Di rencana distribusi fitur distribusi zakat, kita ada rencana men-detailkan sampai ke penerima orang per orang.

Bagian ini belum kita follow up dulu mas.
